myia_backend_relay/myia_backend_relay/relay.py

```diff
@@ -1071,6 +1071,13 @@ def compile(self, graph, argspec, outspec):
             graph, self.context, self.target, self.exec_kind
         )
 
+    def supports_prim_group(self, prim_group):
+        if prim_group is G.universe_operations:
+            # Relay does not explicitly implement primitive make_handle
+            # but still supports universe operations.
+            return True
```

Comment:
That's a brittle check, we should accept any primitive group that contains `make_handle`. A better hardcoded fix would be to write `MAP.get(prim) or prim is P.make_handle` as the condition in the line below. Alternatively, set `P.make_handle: True` in `SIMPLE_MAP`, or have an additional set of supported primitives for Relay.

Reply:
Updated @breuleux with `MAP.get(prim) or prim is P.make_handle` !